### ISSUE TITLE
Contrast of "Files I haven't viewed" ratio too low, hard to read number

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/my-learning-analytics#444]
On the Files I haven't viewed graph bars it's black on a dark grey which is hard to read
![image](https://user-images.githubusercontent.com/27447/54448221-91602180-4721-11e9-810c-4abd44b6803a.png)

Webaim rates this #21252F to #6C757D as 3.27:1
https://webaim.org/resources/contrastchecker/
![image](https://user-images.githubusercontent.com/27447/54448294-bfddfc80-4721-11e9-9c39-3473cead58be.png)

You almost have to get to white to pass the contrast check on this background color, but that doesn't pass the $4682b4 background for "Files I've viewed" 

Maybe we should just use a lighter color like on assignment planning of #e1e1e1

**Comment by jennlove-um** [comment]
The new colors pass web accessibility.  I like that the selected color matches the color used in the Assignment Planning progress bar so that we're keeping the same colors across visualizations.

Testing passes.

**Comment by jonespm** [comment]
Yeah, yeah we really should switch over this project to using sass/scss and then we can define these colors as constants that will be applied through the whole site. Maybe this will be incorporated as part of or after the react changes. This is the what Student Explorer is using and it's pretty convenient.

**Comment by gsilver** [comment]
@jonespm @jennlove-um @pushyamig - looks great.
